### PR TITLE
Update task.py

### PR DESCRIPTION
**Pull request by ntohidi**
Dear Tom, please help in this `loop` topic

### CODE REVIEW

**Comment by tomkidoz** [comment]
All go great

File: task1/task.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #Write a function to print even numbers from 1 to 10
 def printEven():
+    return [x for x in range(1, 10) if x %2 == 0]
```
Comment (tomkidoz) [inline task1/task.py]: do not forget that should be till `11` not `10`

Comment (ntohidi) [reply task1/task.py]: oh ok thx

**Comment by tomkidoz** [comment]
all good merge it

**Pull request merged**